### ISSUE TITLE
`android.support.v8.renderscript` not converted

### ISSUE THREAD

**Issue by robertying** [mikehardy/jetifier#11]
`renderscript` related mappings do not seem to be in `csv` file, since the way to integrate it is different so I guess it is not in the mapping file.

We probably need to add a rule

```diff
- android.support.v8.renderscript.*
+ android.renderscript.*
```

**Comment by mikehardy** [comment]
DO you have an example @robertying ? I need a package to put in https://github.com/mikehardy/rn-androidx-demo so I can reproduce the error and know for sure I fix it

**Comment by robertying** [comment]
https://github.com/react-native-community/react-native-blur

Install this package and there is some lines in `node_modules/@react-native-community/blur/android/src/main/java/com/cmcewen/blurview/BlurringView.java` with `android.support.v8.renderscript.*`

Is this info sufficient for you? I can create a pr to your repro if you have trouble with it. :)

**Comment by mikehardy** [comment]
That's perfect @robertying - thanks https://github.com/react-native-community/react-native-blur/blob/master/android/src/main/java/com/cmcewen/blurview/BlurringView.java#L10

Should be a good test, and the first legitimate jetifier bug yet - it was flawless until now, which never lasts with software :-)

**Comment by robertying** [comment]
I believe this never came in the mapping file was because you only need to add the following:

```gradle
    defaultConfig {
		...
        renderscriptTargetApi 23
        renderscriptSupportModeEnabled true
		...
    }
```

to `app/build.gradle` and you can use `renderscript` with its support library. No need for explicit dependencies.

However, according to this stackoverflow question, people may face some compatibility issue: https://stackoverflow.com/questions/52781821/renderscript-broken-with-androidx. I will test if it is truly broken.

**Comment by mikehardy** [comment]
Thanks again for the report, fixed and released as 1.1.0
